man/liftoffrc.5

```diff
@@ -157,9 +157,9 @@ default:
 Specify the dependency managers to use.
 .Pp
 Currently, this accepts
-.Ic cocoapods
+.Ic cocoapods, carthage
```

Comment:
Ah gotcha. We're off topic but how do you do multiple highlighted words? I guess it doesn't come up that much...